modules/filters/mod_filter.c

```diff
@@ -176,6 +177,14 @@ static int filter_lookup(ap_filter_t *f, ap_filter_rec_t *filter)
                     match = 1;
                     break;
                 }
+                else if (strncmp(*type, r->content_type, sep) == 0
```

Comment:
maybe sep && strncmp ...to be safe here? Even though the source is from the config